templates/quickstart-vpc-for-atlassian-services.yaml

```diff
@@ -244,3 +292,25 @@ Outputs:
     Value: !GetAtt VPCStack.Outputs.NAT2EIP
     Export:
       Name: !Sub '${ExportPrefix}NAT2EIP'
+  VpcTemplateURL:
+    Description: The URL used to source the VPC template
+    Value: !If
+      - NoBucketRegionSuppliedOrPrefixIsDefault
+      - !Sub
+        - https://${QSS3BucketName}.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}submodules/quickstart-aws-vpc/templates/aws-vpc.template
+        - S3Region: !If ["GovCloudCondition", "s3-us-gov-west-1", "s3"]
+      - !Sub
+        - https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}submodules/quickstart-atlassian-services/submodules/quickstart-aws-vpc/templates/aws-vpc.template
+        - S3Bucket: !If [UsingDefaultBucket, !Sub '${QSS3BucketName}-${AWS::Region}', !Ref QSS3BucketName]
+          S3Region: !If [UsingDefaultBucket, !Ref 'AWS::Region', !Ref QSS3BucketRegion]
+  BastionTemplateURL:
+    Description: The URL used to source the Bastion template
+    Value: !If
+      - NoBucketRegionSuppliedOrPrefixIsDefault
+      - !Sub
+        - https://${QSS3BucketName}.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}templates/quickstart-bastion-for-atlassian-services.yaml
+        - S3Region: !If ["GovCloudCondition", "s3-us-gov-west-1", "s3"]
+      - !Sub
+        - https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}submodules/quickstart-atlassian-services/templates/quickstart-bastion-for-atlassian-services.yaml
+        - S3Bucket: !If [UsingDefaultBucket, !Sub '${QSS3BucketName}-${AWS::Region}', !Ref QSS3BucketName]
+          S3Region: !If [UsingDefaultBucket, !Ref 'AWS::Region', !Ref QSS3BucketRegion]  
```

Comment:
Added these for testing purposes. I was planning on removing them but now I'm thinking is there any harm in keeping them? Now that we are moving to regional buckets its might be nice to be able to confirm that the templates are being sourced from where you'd expect them. This info is also useful for us, when debugging issues with production stack issues.